src/Documentation/glossary.js

```diff
@@ -8,16 +8,25 @@ export default {
       name: 'Workspace',
       match: ['workspace'],
       desc: `
-By "workspace" we refer to the directory containing all your project files. For
-example raw datasets, source code, ML models, etc. A workspace becomes a DVC
-project when [\`dvc init\`](/doc/commands-reference/init) is run, and
-[DVC-files](/doc/user-guide/dvc-file-format) are created in it. It\s typically
-also a Git repository.
+Directory containing all your project files. For example raw datasets, source
+code, ML models, etc. A workspace becomes a **DVC project** when
+[\`dvc init\`](/doc/commands-reference/init) is run, and
+[DVC-files](/doc/user-guide/dvc-file-format) are created in it.
+      `
+    },
+    {
+      name: 'DVC Project',
+      match: ['DVC project', 'project', 'projects'],
+      desc: `
+Initialized by running \`dvc init\` in the **workspace**. It will contain the
+\`.dvc/\` directory and [DVC-files](/doc/user-guide/dvc-file-format) created
```

Comment:
same here, this link **https://dvc.org/doc/user-guide/dvc-files-and-directories** is better I think

Reply:
Linked the `.dvc/ directory` part to this in 9206bdf. Is it clearer now?